azure-monitor-opentelemetry-distro/azure/monitor/opentelemetry/distro/constants.py

```diff
@@ -0,0 +1,70 @@
+from os import environ
+from pathlib import Path
+from sys import platform
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+_LOG_PATH_LINUX = "/var/log/applicationinsights"
+_LOG_PATH_WINDOWS = "\\LogFiles\\ApplicationInsights"
+# TODO: Enabled when duplciate logging issue is solved
+# __EXPORTER_DIAGNOSTICS_ENABLED_ENV_VAR = (
+#     "AZURE_MONITOR_OPENTELEMETRY_DISTRO_ENABLE_EXPORTER_DIAGNOSTICS"
+# )
+
+
+def _get_log_path(status_log_path=False):
+    if platform.startswith("linux"):
+        return _LOG_PATH_LINUX
+    elif platform.startswith("win32"):
+        log_path = str(Path.home()) + _LOG_PATH_WINDOWS
+        if status_log_path:
+            return log_path + "\\status"
+        else:
+            return log_path
+    else:
+        return _LOG_PATH_LINUX
+
+
+def _env_var_or_default(var_name, default_val=""):
+    try:
+        return environ[var_name]
+    except KeyError:
+        return default_val
+
+
+def __get_customer_ikey():
+    try:
+        return (
+            environ["APPLICATIONINSIGHTS_CONNECTION_STRING"]
+            .split(";")[0]
+            .split("=")[1]
+        )
+    except KeyError:
+        return "unknown"
+
+
+# TODO: Enabled when duplciate logging issue is solved
+# def __is_exporter_diagnostics_enabled():
+#     return (
+#         __EXPORTER_DIAGNOSTICS_ENABLED_ENV_VAR in environ
+#         and environ[__EXPORTER_DIAGNOSTICS_ENABLED_ENV_VAR] == "True"
+#     )
+
+
+def __is_on_app_service():
+    return "WEBSITE_SITE_NAME" in environ
+
+
+_EXTENSION_VERSION = _env_var_or_default(
```

Comment:
Constants should follow these guidelines:

1. If they are used in more than one place in the code
2. If they do not change within it's lifespan.

Could these values be class variables in `AzureDiagnosticLogger` instead? See [this](https://github.com/microsoft/ApplicationInsights-Python/pull/212/files#r1061971937) comment as well.

Reply:
Done. Left the constants that will be used by both loggers in constants even though I split the pr